### PR TITLE
Desktop-version-portfolio-pull-request

### PR DESCRIPTION
**Pull request by UsamaMasood12**
In this project, I completed the desktop version of the portfolio. I used the mobile-first approach i.e. first I made a mobile version then by alteration and addition of codes, I made a desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -503,3 +523,316 @@ ul.points {
...
+  .containo {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    column-gap: -50px;
+    margin-left: 205px;
+  }
+
+  .container2 {
+    margin-left: -50px;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] According to the Figma design there isn't a See project `button` unless when hovered upon. Kindly fix this 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

- [ ] When one of the cards are hovered on in  your project section your design should look like this ⬇️. Kindly fix this 🙏 

image

File: style.css [hunk 2]
```diff
@@ -382,6 +398,7 @@ ul.container3 {
 .lang {
   font-family: inherit;
   font-size: 20px;
+  color: white;
 }
 
 ul.points {
```
Comment (Ugboaja-Uchechi) [inline style.css:425]: - [ ] kindly make sure the list of languages and skills e.t.c are aligned properly as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 3]
```diff
@@ -16,38 +16,52 @@
...
+  display: none;
 }
 
 .section1 {
-  background-color: black;
+  background-color: #1c1a19;
   width: 375px;
-  margin-top: 120px;
+  margin-top: -30px;
   margin-left: -8px;
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly make sure the content of your design fills the browser viewports width so as to make your design look better and more professional 🙏 

image

File: style.css [hunk 4]
```diff
@@ -503,3 +523,316 @@ ul.points {
...
+    height: 870px;
+  }
+
+  .containe2 {
+    display: none;
+  }
+
+  .im {
+    display: flex;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Your design isn't responsive on all desktop screens. Kindly fix this to make it responsive on all desktop screens 🙏 

image

File: index.html [hunk 5]
```diff
@@ -5,157 +5,190 @@
...
-                            </ul>
+                <div class="aaa">
+                    <div class="aaaa">
+                        <div class="intro">
+                            <div class="gap"></div>
+                            <h1>Hey There. <br>
+                                I'm Usama
+                            </h1>
                         </div>
+                        <div class="profession"><h2>I'am a Software Developer</h2></div>
```
Comment (Ugboaja-Uchechi) [inline index.html:42]: - [ ] Great job 👍 You gave this text a `font-family` of `Roboto` instead of `Inter`. Kindly fix this 🙏

File: index.html [hunk 6]
```diff
@@ -5,157 +5,190 @@
...
-                        <div class="box box1"><img src="./image/44.png" alt=""></div>
+                    <div class="containe">
+                        <div class="containe1">
+                            <div class="aaaaa">
+                                <article class="services">
+                                    <p>
+                                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                                        don’t hestiate to contact me.
+                                    </p>
+                                </article>
```
Comment (Ugboaja-Uchechi) [inline index.html:52]: - [ ] Great job 👍 You gave this text a `font-family` of `Poppins` instead of `Inter`. Kindly fix this 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 7]
```diff
@@ -503,3 +544,354 @@ ul.points {
...
+  .header {
+    margin-top: 7px;
+    margin-left: 80px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .body1 {
```
Comment (rskinnerc) [inline style.css:599]: ♻️ Great job using media queries as required!! However, please note that once the viewport width hits the min-width of 768px, the design totally breaks. There is a lot of white space to the right and most of the content is being pushed to the left.

The following screenshot was taken in a 768px width viewport. Please note how there are elements pushed to the left and the content remains to the right. Also not the bottom scrollbar, it should become responsive. That is, the width should fit the full viewport width so it doesn't break or requires the user to scroll horizontally.

image


- [ ] Please make sure that the design fits responsively in all screen sizes. hat would make your design more professional.

This is also happening on some mobile devices. You can check on your mobile the live version [Here](https://raw.githack.com/UsamaMasood12/Portfolio-finish-mobile-version/Desktop-Version/index.html)

File: style.css [hunk 8]
```diff
@@ -503,3 +544,354 @@ ul.points {
...
+  }
+
+  .aaa {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .aaaaa {
+    display: flex;
+    flex-direction: row-reverse;
```
Comment (rskinnerc) [inline style.css]: [OPTIONAL] ⭐ This is an optional recommendation. Please consider using more meaningful names in your classes. That would make your work more professional, readable, and easy to maintain. Whenever you have time, kindly take a look at this short article. That could be a good option [CSS Classes naming conventions](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)

File: index.html [hunk 9]
```diff
@@ -5,157 +5,190 @@
...
+                            <div class="hovee">
+                                <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+                                <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+                                <ul class="container3">
+                                    <li class="item4">css</li>
+                                    <li class="item5">bootstrap</li>
+                                    <li class="item6">Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="container4">
```
Comment (rskinnerc) [inline index.html:114]: [OPTIONAL] ⭐ This is an optional change but recommended. I strongly suggest that you keep your code as simplest as possible. Wrapping other elements like `h1` or `p` tags just for styling into a `div` is not recommended. This makes your code less readable, not very semantic and it is not screen-reader friendly. Remember that you always can style those tags directly.

Please also remember that thee `h`1` tag is the most important title on your page. You should avoid using several of these.

**Review by rskinnerc — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12 👋👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy


 ## Highlights
- You have configured the linters correctly and they are passing ✅
- You are using GitHub Flow correctly
- Good job implementing the hover effect to the project cards 😜😜

 ## Required Changes ♻️
_Check the comments under the review._

Good job implementing some of the required changes. Please check my comments below though. There are still some changes you need to work on. You did a good job starting with the mobile-first approach. From there, once the viewport hits the required `min-width`, it should fit correctly up to the widest resolution (think about how to make the width fluid inside the media query). Kindly, review my required changes as well as double-check the previous ones. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -5,157 +5,190 @@
...
+                                </div>
+                        </div>
+                        <div class="container2">
+                            <div class="hovee">
+                                <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+                                <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+                                <ul class="container3">
+                                    <li class="item4">css</li>
+                                    <li class="item5">bootstrap</li>
+                                    <li class="item6">Ruby</li>
```
Comment (DuaneDave) [inline index.html:111]: [OPTIONAL]  I suggest that you fix how the grid behaves from `768px` to ensure a consistent design on all breakpoints.
[see here](https://duanedave.github.io/Portfolio-Website/) play with the responsiveness and see.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✅ Great UI 👍 
✅ Meaningful commit history 👍 


 ## Required Changes ♻️
- As requested by the previous reviewers, kindly implement this section at least to look like the figma design. Currently the illustration is not fitting properly from `768px` break point.
     
     -TIP : From my observation i noticed that the text size is not responsive and also the space between the social media icons and text should be adjusted. Finally play a little with the width and height of the illustration

Your design 👇👇👇

badddddddd


Figma design 👇👇👇

gooooood
  

_Check the comments above the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @UsamaMasood12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Nice UI design :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**